connect-examples/v1/java/src/main/java/com/squareup/connectexamples/PaymentsReporter.java

```diff
@@ -206,10 +217,14 @@ public void printSalesReport(List<JSONObject> payments) {
 
   // Call the methods defined above
   public static void main(String[] args) {
-    PaymentsReporter reporter = new PaymentsReporter();
-    List<JSONObject> payments = reporter.get2015Payments(reporter.getLocationIds());
-    if (payments != null) {
-      reporter.printSalesReport(payments);
+    try {
```

Comment:
why are you doing try & catch here? What error did you see locally?

Reply:
get2015Payments (which I will definitely rename now lol...) can throw an exception, so it must be in try block. The most common error I'm catching here is if they just forget to replace the credentials. Let me know if that doesn't make sense or if I'm doing it incorrectly, though!

Reply:
okay that makes sense!